contracts/Comptroller/Diamond/facets/SetterFacet.sol

```diff
@@ -324,7 +324,7 @@ contract SetterFacet is FacetBase {
      * @param paused_ The new paused state (true=paused, false=unpaused)
      */
     function _setActionsPaused(address[] calldata markets_, Action[] calldata actions_, bool paused_) external {
-        ensureAllowed("_setActionsPaused(address[],uint256[],bool)");
```

Comment:
If we do this we should authorized the Normal Timelock in the ACM to execute this new function. We should do this in the VIP associated to this deployment

Reply:
I have updated the VIP for the diamond proxy: https://github.com/VenusProtocol/vips/pull/11/commits/6cdc98b1677e6e0c8abfdc2726515b4e2063abfd